src/etl/ETLSource.cpp

```diff
@@ -673,6 +674,8 @@ class AsyncCallData
                         request_.ledger().sequence(),
                         std::string{obj.key()});
                 lastKey_ = obj.key();
+                backend.writeNFTs(std::move(getNFTDataFromObj(
```

Comment:
I think we should probably check whether this is an nft token dir before calling this function. Look at `isOfferDir` as an example

Reply:
why @cjcobb23? that check happens within the `getNFTDataFromObj` function, which will return an empty vector if it's not an NFT page. then `writeNFTs` will not loop over anything